systers_portal/meetup/models.py

```diff
@@ -2,122 +2,59 @@
-            ('approve_meetup_location_meetuprequest', 'Approve meetup location meetup request'),
-            ('reject_meetup_location_meetuprequest', 'Reject meetup location meetup request'),
-            ('view_meetup_location_meetuprequest', 'View meetup location meetup request'),
-            ('approve_meetup_comment', 'Approve comment for a meetup'),
-            ('reject_meetup_comment', 'Reject comment for a meetup'),
-            ('add_meetup_rsvp', 'RSVP for a meetup'),
-            ('approve_support_request', 'Approve support request'),
-            ('reject_support_request', 'Reject support request'),
-            ('add_support_request_comment', 'Add comment for a support request'),
-            ('edit_support_request_comment', 'Edit comment for a support request'),
-            ('delete_support_request_comment', 'Delete comment for a support request'),
-            ('approve_support_request_comment', 'Approve comment for a support request'),
-            ('reject_support_request_comment', 'Reject comment for a support request')
-        )
-
-    def __str__(self):
-        return self.name
-
-
-class RequestMeetupLocation(models.Model):
-    """Manage details of Meetup Location requests"""
-    name = models.CharField(max_length=255, unique=True, verbose_name="Name of the Meetup Location."
-                            "(Naming convention for Systers meetup location is City+Systers."
-                            "e.g.London Systers, Boston Systers.)")
-    slug = models.SlugField(max_length=150, unique=True,
-                            verbose_name="Slug of the Meetup Location")
-    location = models.ForeignKey(City, verbose_name="Location")
-    description = RichTextField(
-        verbose_name="Description of the Meetup Location")
-    email = models.EmailField(max_length=255, blank=True,
-                              verbose_name="Email of the Meetup Location if any.")
-    user = models.ForeignKey(
-        SystersUser, related_name='createdby', verbose_name="Requested By")
-    approved_by = models.ForeignKey(SystersUser, blank=True, null=True,
-                                    related_name='approvedby', verbose_name="Approved By")
-    date_created = models.DateTimeField(
-        auto_now_add=True, verbose_name="Date Created")
-    is_approved = models.BooleanField(
-        default=False, verbose_name="Is this Approved?")
-
-    def get_verbose_fields(self):
-        """Get verbose names of RequestMeetupLocation object's model fields
-        :return: list of tuples (verbosefieldname, fieldvalue)
-        """
-
-        return [(field.verbose_name, getattr(self, field.name)) for field in
-                RequestMeetupLocation._meta.fields]
-
-    def __str__(self):
-        return self.name
-
-
 class Meetup(models.Model):
     """Manage details of Meetups of MeetupLocations"""
-    title = models.CharField(max_length=50, verbose_name="Title",)
+    title = models.CharField(max_length=50, verbose_name="Title", )
```

Comment:
Lets get this PR merged! I ran it locally and it works fine for me. You haven't made these changes but its minor.